### DISCUSSION TITLE
I made a new GNOME Extension to quickly select presets

### DISCUSSION THREAD

**Discussion by ulville** [wwmm/easyeffects#1438] [Show and tell]
Hi dear EasyEffects devs and users :raised_hands: 

I'm the author of [this](https://github.com/ulville/eepresetselector) new **EasyEffects Preset Selector GNOME Extension**. I am so happy to announce that it's ready and fully compatible with both Flatpak and and non-flatpak versions of EasyEffects now :tada: 

It basicly adds a button to your top panel. Then you can click on it to see your available presets and click one to activate it without opening EasyEffects window. I believe it can serve well for a lot of people's use cases. Please give it a try and let me know what you think about it. Hope you enjoy it!

Feel free to contrubute by testing, opening issues or with code. And feel free to tell the others about it who you think may enjoy it.

You can install the latest version directly from [GNOME Extensions website](https://extensions.gnome.org/extension/4907/easyeffects-preset-selector/)

![Extension](https://github.com/ulville/eepresetselector/blob/master/screenshots/screenshot.png?raw=true)

### Why?

I use different presets for listening music, watching movies, playing games, watching people talking on youtube etc. And I find it a bit tedious to open EasyEffects app and select the preset I want everytime I wanted to change it. (using command-line options are also not so quick and user-friendly either) So I felt the need of something that will let me select the preset I want quickly from my GNOME Shell's top panel. I searched for a program or extension to let me do that but surprisingly there was nothing. So I decided to create it myself (Being in a quarantine for a week helped a lot, thx cov-19)

**Comment by wwmm** [comment]
Thanks @ulville! A long time ago someone suggested that. But besides the fact I have never written a GNOME extension before I Don't really have the time to maintain one. So it is good to see a third party project filling that gap. Feel free to create an entry in our wiki for it.

**Reply by ulville** [reply]
Thank you for creating such a useful software! I've taken a lot from the open source community, it's nice to be able to give a little something back. I added the page to the wiki  :+1:
